Documentation/articles/content/adding_ttf_fonts.md

```diff
@@ -7,13 +7,11 @@ MonoGame supports more than one method of using fonts, the following is an expla
 To be able to use a TrueType font, MonoGame requires the TrueType font file and a .spritefont file.
 TrueType fonts may be installed on the system, or added manually in the same directory as the .spritefont file.
 
-1. Create the .spritefont file.
+1. Create the .spritefont file by selecting "Edit -> Add -> New Item" from the MGCB Editor menu, then select **SpriteFont Description** from the list and click **Create**.
 
-  ![Adding TTF fonts step 1](~/images/adding_ttf_fonts_step_1.PNG)
+  ![Adding TTF fonts step 2](~/images/content/adding_ttf_fonts.PNG)
 
-  ![Adding TTF fonts step 2](~/images/adding_ttf_fonts_step_2.PNG)
-
-1. Open the newly created .spritefont file in your text editor of choice, find this line and change it to your selected .ttf font.
+2. Open the newly created .spritefont file in your text editor of choice, find this line and change it to your selected .ttf font.
```

Comment:
Please leave this at 1. The markdown renderer automatically sets the right numbers in numbered lists, so conventions often use 1 for all entries in a list so things can be reordered/added/deleted without issue. We should probably have a style guide. I use the MS doc extension for vs code which is great to standardize formatting in md files and detect issues.

You can see the list is correct in the rendered file: https://docs.monogame.net/articles/content/adding_ttf_fonts.html

Reply:
Apologies for the delay, sorted now @Jjagg